src/bug-filing/common/issue-filing-url-string-utils.ts

```diff
@@ -30,19 +29,26 @@ const getSelectorLastPart = (selector: string): string => {
     return selectorLastPart;
 };
 
-const appendSuffixToUrl = (url: string, suffix: string): string => {
-    if (endsWith(url, suffix) || endsWith(url, `${suffix}/`)) {
-        return url;
+const appendIssuesSuffixToGitHubUrl = (url: string): string => {
+    const ownerAndRepo = new RegExp('^https?://github.com/[^/]+/[^/]+$');
```

Comment:
I think this prefix check is too strict; this doesn't support valid github.com urls like "www.github.com" (I'm not sure if there are any other) and also doesn't support enterprise github endpoints even though I'd expect we otherwise could

Reply:
Updated